modernrpc/apps.py

```diff
@@ -83,3 +84,13 @@ def import_modules(modules_list: List[str]) -> None:
                 # And register only functions with attribute 'modernrpc_enabled' defined to True
                 if getattr(func, "modernrpc_enabled", False):
                     registry.register_method(func)
+
+    @staticmethod
+    def secure_xmlrpc_use():
+        try:
+            import defusedxml.xmlrpc
+        except ImportError:
+            logger.debug("'defusedxml' package were not found, no protection will be enabled against XML based attacks")
+            return
+        else:
+            defusedxml.xmlrpc.monkey_patch()
```

Comment:
The `else` clause is redundant, see the comments from Codacy .

Reply:
I think you can move this line immediately after line 91 where defusedxml is imported but that's more of a reference choice.